### DISCUSSION TITLE
G2Plot 图例的 marker 的实心空心是什么决定的

### DISCUSSION THREAD

**Discussion by tobemaster56** [antvis/G2#5155] [Q&A]
legend.marker.symbol 决定了 marker 的形状，有些图形下 marker 是实心填充的，有些图形又是空心的，有配置项决定吗

**Comment by hustcc** [comment]
有 图形的 shape 决定的。

**Reply by tobemaster56** [reply]
举个实际的例子吧，
https://codesandbox.io/s/blazing-monad-4ydruh
雷达图中，我调整了 legend.marker.symbol 为 square，但是是空心的效果，如果期望是实心的效果，该怎么调整
